pkg/components/stateful_set.go

```diff
@@ -105,7 +105,7 @@ func (s *StatefulSet) AddPod(obj *Pod) error {
 	o := obj.GetObj()
 	s.obj.TemplateMetadata = &o.PodTemplateMeta
 	s.obj.PodTemplate = o.PodTemplate
-
+	s.AddMatchLabelsSelectors(o.PodTemplateMeta.Labels)
```

Comment:
Is it only for stateful set or it is applicable for other objects like deployment, daemon set etc...?